staging/src/k8s.io/controller-manager/go.mod

```diff
@@ -5,16 +5,24 @@ module k8s.io/controller-manager
 go 1.15
 
 require (
+	github.com/spf13/pflag v1.0.5
+	github.com/stretchr/testify v1.4.0
 	k8s.io/apimachinery v0.0.0
 	k8s.io/apiserver v0.0.0
+	k8s.io/client-go v0.0.0
+	k8s.io/cloud-provider v0.0.0
 	k8s.io/component-base v0.0.0
+	k8s.io/klog/v2 v2.2.0
+	k8s.io/kube-controller-manager v0.0.0
```

Comment:
these two dependencies should not exist... I expected cloud-provider and kube-controller-manager to depend on the generic controller-manager module, not the other way around

Reply:
A little uncomfortable that we may have put too much (interface definition and controllers) into cloud-provider. Topic for another time.